app/views/support_interface/application_forms/nationalities/edit.html.erb

```diff
@@ -0,0 +1,41 @@
+<% content_for :browser_title, title_with_error_prefix('Edit applicant nationality', @nationalities_form.errors.any?) %>
+<% content_for :before_content, govuk_back_link_to(support_interface_application_form_path(@application_form)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @nationalities_form, url: support_interface_application_form_edit_nationalities_path, class: 'app-nationality', method: :patch do |f| %>
+      <%= f.govuk_error_summary %>
+
+      <%= f.govuk_check_boxes_fieldset :nationalities, legend: { size: 'l', text: 'Edit applicant nationality' } do %>
+        <%= f.govuk_check_box(
+          :nationalities,
+          'British',
+          link_errors: true,
+          multiple: true,
+          label: { text: 'British' },
```

Comment:
There's a mixture of using localizations and hardcoded strings for labels in this file, does that matter?

Reply:
Not especially, IMHO